src/pages/api/download-reports/funders.ts

```diff
@@ -0,0 +1,72 @@
+import { NextApiRequest, NextApiResponse } from "next"
+import { gql } from '@apollo/client';
+import apolloClient from '../../../utils/apolloClient'
+import { Facet } from "src/components/FacetList/FacetList";
+
+const QUERY = gql`
+  query getOrganizationQuery(
+    $id: ID
+    $gridId: ID
+    $crossrefFunderId: ID
+    $cursor: String
+    $filterQuery: String
+    $published: String
+    $resourceTypeId: String
+    $fieldOfScience: String
+    $language: String
+    $license: String
+    $registrationAgency: String
+  ) {
+    organization(
+      id: $id
+      gridId: $gridId
+      crossrefFunderId: $crossrefFunderId
+    ) {
+      works(
+        first: 200
+        after: $cursor
+        query: $filterQuery
+        published: $published
+        resourceTypeId: $resourceTypeId
+        fieldOfScience: $fieldOfScience
+        language: $language
+        license: $license
+        registrationAgency: $registrationAgency
+      ) {
+        funders {
```

Comment:
Right now this would return the default number of facets....it looks like works within an organization do not allow you to specify the number of facets to be returned.  This is an oversight in the current graphql api.

Reply:
This will need to wait on the backend to be updated to allow facetCount for organization works

Reply:
The api now accounts for this in [this PR](https://github.com/datacite/lupo/pull/994)